data/models/canteen.py

```diff
@@ -58,7 +58,11 @@ class PublicationStatus(models.TextChoices):
     daily_meal_count = models.IntegerField(
         null=True, blank=True, verbose_name="repas par jour"
     )
+    # TODO: index by siret if it exists?
```

Comment:
Would be good once we know the SIRET format we will use (and we have them all normalized)